config/service.go

```diff
@@ -0,0 +1,13 @@
+package config
+
+import (
+	"github.com/buildwithgrove/path/relayer"
+)
+
+const (
```

Comment:
Nice, I prefer this implementation. Simplifies the config YAML and makes it easy to find which service uses which qos.